utils/installer/lv-config.example.lua

```diff
@@ -66,6 +66,24 @@ O.lang.php.diagnostics.signs = true
 O.lang.php.diagnostics.underline = true
 O.lang.php.filetypes = {"php", "phtml"}
 
+--LaTeX
+O.lang.latex.aux_directory = "."
+O.lang.latex.bibtex_formatter = "texlab"
+--TODO build args are hardcoded; args are not passed through to ftplugin/tex.lua
```

Comment:
To update

Reply:
Sorry. I forgot to update the build args.